data/json/snippets/mutant_anatomy.json

```diff
@@ -43,6 +43,182 @@
       }
     ]
   },
+  {
+    "type": "snippet",
+    "category": "mutant_brain_desc",
+    "text": [
+      {
+        "id": "mutant_brain_1",
+        "text": "The brain of a heavily mutated animal.  A thick net of interlocking veins coccoons it, and the surface has a rainbow sheen like an oil slick.  When you move to touch it the whole thing starts shuddering and it comes loose from the spinal cord, like it's trying to escape."
+      },
+      {
+        "id": "mutant_brain_2",
+        "text": "The brain of a heavily mutated animal.  Shining white in color, and brittle to the touch like hard cheese you can hardly believe it was a living organ moments before.  On the base the pure whiteness is broken up by a huge, spastic artery filled with yellow gunk."
+      },
+      {
+        "id": "mutant_brain_3",
+        "text": "The brain of a heavily mutated animal.  Vestigal, still-twitching eyes are strewn across its surface, each with a different pupil.  Hidden between the hemispheres you find an embryonic copy of the creature's face, eyes darting around in a mad spasm."
+      },
+      {
+        "id": "mutant_brain_4",
+        "text": "What probably was the brain of a heavily mutated animal.  The spinal cord gives way to a chaotic tangle of loose neural fibers without any of semblance of a solid anatomy, and the whole mess is visibly shrinking now that you got it out of the creature's liquor."
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "cooked_mutant_brain_desc",
+    "text": [
+      {
+        "id": "cooked_mutant_brain_1",
+        "text": "This is a piece of cooked brain from a mutated animal.  Whatever gave its surface the oily sheen coagulated into bluish clumps of goo settled into the gyri.  You try to scrape them off, but they seem fused with the tissue beneath."
+      },
+      {
+        "id": "cooked_mutant_brain_2",
+        "text": "A piece of cooked brain from a heavily mutated animal.  A vein at the base of it burst while cooking, slathering the whole thing in faintly citrus-smelling yellow liquid.  While the smell would be almost appetizing, the faint taste of ammonia is anything but."
+      },
+      {
+        "id": "cooked_mutant_brain_3",
+        "text": "A piece of cooked brain from a mutant.  The eyes jutting out of the surface have grown milky, and a good few of them fell off, each revealing a small depression dominated by a short root of veins and nerve.  Judging by the spongy texture, some remain buried in the deeper tissue."
```

Comment:
I didn't want to mirror the uncooked ones 1:1, mostly because there's no way to match a raw description to a cooked one. Figured omitting some extra eyes made sense even if you got a different raw brain originally, but a suddenly appearing face would be a bit more jarring.